tests/integration/components/bulma-radio-test.js

```diff
@@ -1,14 +1,49 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
+import { find, click } from 'ember-native-dom-helpers';
+
+const sampleRadioButtons = hbs`
+  {{#bulma-radio name="test" value="cool" id="test-cool"}}
+    Cool
+  {{/bulma-radio}}
+
+  {{#bulma-radio name="test" value="awesome" id="test-awesome"}}
+    Awesome
+  {{/bulma-radio}}
+`;
 
 moduleForComponent('bulma-radio', 'Integration | Component | bulma radio', {
   integration: true
 });
 
-test('it renders', function(assert) {
-  // Set any properties with this.set('myProperty', 'value');
-  // Handle any actions with this.on('myAction', function(val) { ... });
-  this.render(hbs`{{bulma-radio}}`);
+test('it renders radio buttons with labels', function(assert) {
+  this.render(sampleRadioButtons);
+
+  assert.ok(find('#test-cool').textContent.trim(), 'Cool');
+  assert.ok(find('#test-awesome').textContent.trim(), 'Awesome');
+});
+
+test('it selects an option', async function(assert) {
+  this.render(sampleRadioButtons);
+
+  await click('#test-cool');
```

Comment:
@jbailey4 Maybe trigger click a second time and assert the opposite to ensure that the `name` attribute is working properly.